tester/tests/coco_edge_cases

```diff
@@ -1,5 +1,48 @@
 echo "COCO TEST 🥥"
 
+echo RUNNING: 'echo -'
+echo -
+echo
+
+echo RUNNING: 'echo -n'
+echo -n
+echo
+
+echo RUNNING: 'echo -nn'
+echo -nn
+echo
+
+echo RUNNING: 'echo -nn -n'
+echo -nn -n
+echo
+
+echo RUNNING: 'echo -nna -n'
+echo -nna -n
+echo
+
+echo RUNNING: 'sleep 2 | ls'
+sleep 2 | ls
+echo
+
+echo RUNNING: 'echo 4 | cat - should print 4 right away'
+echo 4 | cat
+echo
+
+echo RUNNING: 'mkdir testdir && chmod 111 testdir/ && ls'
```

Comment:
you need to cd into the folder testdir before the ls... and then cd .. for the next test